src/getstream/updateBetterChannelMembers.js

```diff
@@ -45,25 +46,35 @@ const __getAnonUserInfoPriority = (memberDataFromDb, member) => {
       emoji_name: memberDataFromDb.anon_user_info_emoji_name,
       emoji_code: memberDataFromDb.anon_user_info_emoji_code
     };
-  } else if (member.anon_user_info_color_code) {
+  }
+  if (member.anon_user_info_color_code) {
     return {
       color_name: member.anon_user_info_color_name,
       color_code: member.anon_user_info_color_code,
       emoji_name: member.anon_user_info_emoji_name,
       emoji_code: member.anon_user_info_emoji_code
     };
-  } else {
-    const emoji = BetterSocialConstantListUtils.getRandomEmoji();
-    const color = BetterSocialConstantListUtils.getRandomColor();
-    return {
-      color_name: color.color,
-      color_code: color.code,
-      emoji_name: emoji.name,
-      emoji_code: emoji.emoji
-    };
   }
+  const emoji = BetterSocialConstantListUtils.getRandomEmoji();
+  const color = BetterSocialConstantListUtils.getRandomColor();
+  return {
+    color_name: color.color,
+    color_code: color.code,
+    emoji_name: emoji.name,
+    emoji_code: emoji.emoji
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code seems to be repeating itself. The same structure is returned in two different if conditions. Consider refactoring this to avoid repetition and improve maintainability.

```diff
-  if (memberDataFromDb.anon_user_info_color_code) {
-    return {
-      color_name: memberDataFromDb.anon_user_info_color_name,
-      color_code: memberDataFromDb.anon_user_info_color_code,
-      emoji_name: memberDataFromDb.anon_user_info_emoji_name,
-      emoji_code: memberDataFromDb.anon_user_info_emoji_code
-    };
-  }
-  if (member.anon_user_info_color_code) {
-     return {
-       color_name: member.anon_user_info_color_name,
-       color_code: member.anon_user_info_color_code,
-       emoji_name: member.anon_user_info_emoji_name,
-       emoji_code: member.anon_user_info_emoji_code
-     };
-   }

+  const userInfoSource = memberDataFromDb.anon_user_info_color_code ? memberDataFromDb : member;
+  if (userInfoSource.anon_user_info_color_code) {
+    return {
+      color_name: userInfoSource.anon_user_info_color_name,
+      color_code: userInfoSource.anon_user_info_color_code,
+      emoji_name: userInfoSource.anon_user_info_emoji_name,
+      emoji_code: userInfoSource.anon_user_info_emoji_code
+    };
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->